mgmtsystem/security/ir.model.access.csv

```diff
@@ -1,5 +1,10 @@
-"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
-"access_mgmtsystem_system_user","mgmtsystem.system.user","model_mgmtsystem_system","mgmtsystem.group_mgmtsystem_user",1,0,0,0
-"access_mgmtsystem_system_viewer","mgmtsystem.system.viewer","model_mgmtsystem_system","mgmtsystem.group_mgmtsystem_viewer",1,0,0,0
-"access_mgmtsystem_system_auditor","mgmtsystem.system.auditor","model_mgmtsystem_system","mgmtsystem.group_mgmtsystem_auditor",1,0,0,0
-"access_mgmtsystem_system_manager","mgmtsystem.system.manager","model_mgmtsystem_system","mgmtsystem.group_mgmtsystem_manager",1,1,1,1

```

Comment:
I see something weird too related with EOL but I don't know what it is.

Reply:
Usually windows puts a CRLF end, and Linux a LF end. If your editor is programmed to fix EOLs and the original author developed this with CRLF, it can be a problem.

Well, I used github's feature to ignore whitespace changes to see this diff, so don't worry too much about it.